tests/acceptance/common.py

```diff
@@ -222,13 +222,16 @@ def determine_active_passive_part(bitbake_variables, conn):
         )
 
 
-def get_ssh_common_args():
-    return "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+def get_ssh_common_args(conn):
+    args = "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+    if "key_filename" in conn.connect_kwargs.keys():
+        args += " -i %s" % conn.connect_kwargs["key_filename"]
```

Comment:
I would add a space here, just in case, i.e.: `" -i %s"` -> `" -i %s "`

Reply:
also maybe some quotes around the %s? as in: what if the `conn.connect_kwargs["key_filename"]` contains spaces?

Reply:
Why the space? We know exactly how and where is this used. Similarly for the quotes, it is only used in one place and we know the path.

Generally speaking, when I write test code I try not to be too defensive and let things fail. A failure might hint someone that is doing something fishie.